api/src/main/java/edu/cornell/mannlib/vitro/webapp/imageprocessor/imageio/IIOImageProcessor.java

```diff
@@ -55,8 +54,8 @@
 	 */
 	@Override
 	public void startup(Application application, ComponentStartupStatus ss) {
-		JAI.getDefaultInstance().setImagingListener(
-				new NonNoisyImagingListener());
+//		JAI.getDefaultInstance().setImagingListener(
```

Comment:
Please remove, instead of commenting out.

Reply:
done